admin-dev/themes/new-theme/js/pages/country/index.ts

```diff
@@ -52,5 +54,9 @@ $(() => {
   countryGrid.addExtension(new FiltersSubmitButtonEnablerExtension());
   countryGrid.addExtension(new ColumnTogglingExtension());
 
```

Comment:
by the way. I think we used to create a dedicated javascript file (and entry) for form. (one for index page, other for form), so that we don't include unused js code in index page and vice versa.

So i think all components for form should be moved to a adedicated form.ts or so and then the new entry should be included in form js block. With that said I don't think you need to introduce the "zip-code-manager" class, and can just init the toggler component in that fresh form.js